metadeploy/api/salesforce.py

```diff
@@ -0,0 +1,260 @@
+"""
+Salesforce utilities
+"""
+
+
+import json
+import os
+from datetime import datetime
+from unittest.mock import Mock
+
+from cumulusci.core.config import OrgConfig, TaskConfig
+from cumulusci.core.runtime import BaseCumulusCI
+from cumulusci.oauth.salesforce import SalesforceOAuth2, jwt_session
+from cumulusci.tasks.salesforce.org_settings import DeployOrgSettings
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from simple_salesforce import Salesforce as SimpleSalesforce
+
+# Salesforce connected app
+# Assign these locally, for brevity:
+SF_CALLBACK_URL = settings.CONNECTED_APP_CALLBACK_URL
+SF_CLIENT_KEY = settings.CONNECTED_APP_CLIENT_KEY
+SF_CLIENT_ID = settings.CONNECTED_APP_CLIENT_ID
+SF_CLIENT_SECRET = settings.CONNECTED_APP_CLIENT_SECRET
+SFDX_SIGNUP_INSTANCE = settings.SFDX_SIGNUP_INSTANCE
+
+DURATION_DAYS = 30
+
+
+def _get_devhub_api():
+    """Get an access token.
+
+    Get an access token (session) using the global dev hub username.
+    """
+    if not settings.DEVHUB_USERNAME:
+        raise ImproperlyConfigured(
+            "You must set the DEVHUB_USERNAME to connect to a Salesforce organization."
+        )
+    jwt = jwt_session(SF_CLIENT_ID, SF_CLIENT_KEY, settings.DEVHUB_USERNAME)
+    return SimpleSalesforce(
+        instance_url=jwt["instance_url"],
+        session_id=jwt["access_token"],
+        client_id="MetaDeploy",
+        version="47.0",
```

Comment:
```suggestion
        version="49.0",
```

(Updating to current version of the API)